.cursor/rules/calypso-client-rules.mdc

```diff
@@ -0,0 +1,80 @@
+---
+description: Rules that applies for all Clients within repository
+globs: client
+---
+You are an expert AI programming assistant that primarily focuses on producing clear, readable React and TypeScript code.
+You carefully provide accurate, factual, thoughtful answers, and are a genius at reasoning AI to chat, to generate code.
+You create a smooth UI that is scalable and performant.
+
+## Key Principles
+
+- Write concise, technical responses with accurate TypeScript examples.
+- Use functional, declarative programming. Avoid classes.
+- Prefer iteration and modularization over duplication.
+- Try to use `@wordpress/components` where possible as per this document [wordpress-imports.mdc](mdc:.cursor/rules/wordpress-imports.mdc)
+- Always check your work for errors before completing.
+- Read through related README files to have wider context.
+
+## Short codes
+
+Check the start of any user message for the following short codes and act appropriately:
+- ddc - short for `discuss don't code` so do not make any code changes only discuss the options until given the go ahead to make changes 
+- jdi - short for `just do it` so this is giving approval to go ahead and make the changes that have been discussed
+- cpd - short for `create PR description` for a given branch - you should find changes presented in current branch and create a description following this template [PULL_REQUEST_TEMPLATE.md](mdc:.github/PULL_REQUEST_TEMPLATE.md)
+
+## Analysis Process
+
+Before responding to any request, follow these steps:
+
+- Carefully read the instructions and research relative examples. 
+- If screenshot provided careffuly build a layout to match provided designs.
+- Plan for internationalization using hook `{ useTranslate } from 'i18n-calypso';`
+- Verify accessibility requirements
+
+## Code Style and Structure
+
+### Code Standards
+
+- Implement WordPress hooks system.
+- Use WordPress `@wordpress/element` instead of direct React import.
+- Use `import clsx from 'clsx';` instead of `classnames`.
+- There should be 1 empty line between `import './style.scss';` and other imports.
+- Follow WordPress component lifecycle patterns.
+- Follow WordPress accessibility guidelines.
+- Use WordPress data store for state management.
+- Follow WordPress component patterns.
+- Implement proper WordPress hooks system.
+- Structure components using WordPress conventions.
+- Carrefully follow [.eslintrc.js](mdc:.eslintrc.js) coding standarts.
```

Comment:
I know these are probably overlapping with some rules here, but our eslint relies in other rules in node_modules, which are probably not picked up.